resources/views/home.blade.php

```diff
@@ -1,5 +1,322 @@
+                    <p class="text-xl lg:text-2xl font-light text-center lg:text-start">
+                        {{__("content.section_1.subtitle_1")}}
+                        <span class="font-medium">{{__("content.section_1.subtitle_2")}}</span>
+                    </p>
+                    <div class="my-6">
+                        <x-primary-button href="">{{ __("content.pages.about") }}</x-primary-button>
+                    </div>
+                </div>
+            </div>
+
+
+            <div class="flex basis-1/2 relative self-center">
+                <img src="graphics/web-1.jpg" class="h-auto w-min aspect-auto" alt="placeholder"/>
+            </div>
+        </section>
+
+        <section class="text-center section-padding">
+            <div class="text-2xl lg:text-3xl font-medium text-gray-400">{{ __("content.section_2.title_1") }}<span
+                    class="text-website-normal">{{ __("content.section_2.title_2") }}</span>
+            </div>
+
+            <div class="grid grid-cols-2 md:flex md:flex-row pb-12 text-gray-400 items-center justify-center mt-6">
+                <div class="m-6">
+                    <img src="images/clients/cwup.png" class="client-icon" alt="Collegium Witelona"/>
+                </div>
+                <div class="m-6">
+                    <img src="images/clients/insly.png" class="client-icon" alt="Insly"/>
+                </div>
+                <div class="m-6">
+                    <img src="images/clients/kghm.png" class="client-icon" alt="KGHM"/>
+                </div>
+                <div class="m-6">
+                    <img src="images/clients/vitaplus.png" class="client-icon" alt="Vita Plus"/>
+                </div>
+            </div>
+        </section>
+
+        <section class="bg-gradient-to-r from-gray-light to-white">
+
+            <x-waves.wave-1.top></x-waves.wave-1.top>
+            <div class="flex flex-col lg:flex-row section-padding lg:pr-0 pt-24">
+                <div
+                    class="flex basis-2/5 flex-col lg:pr-8 delay-300 duration-700 taos:translate-x-48 taos:opacity-0"
+                    data-taos-offset="100">
+                    <h2 class="text-3xl md:text-4xl lg:text-6xl font-semibold text-gray-900 mb-4 text-center md:text-left">
+                        {{__("content.section_3.title_1")}}</h2>
+                    <div class="text-black subtitle text-center md:text-left py-6">
+                        <div class="pb-4 md:pb-0"> {{__("content.section_3.subtitle_1")}}
+                            <span class="font-medium">
+                                {{__("content.section_3.subtitle_2")}}
+                                </span>
+                        </div>
+                        <ul class="py-2 md:-ml-4 text-md justify-center">
+                            <li class="flex items-center text-left">
+                                <x-icons.arrow color="#5379f6"></x-icons.arrow>
```

Comment:
Maybe create custom color for that `#5379f6` in config.

Reply:
deleted color in component, it's a gif now, no color needed